v3/swagger/gen/scylla-manager/models/restore_progress.go

```diff
@@ -23,7 +23,7 @@ type RestoreProgress struct {
 	// Format: date-time
 	CompletedAt *strfmt.DateTime `json:"completed_at,omitempty"`
 
-	// downloaded
+	// This field is DEPRECATED. Total bytes downloaded from table (included in restored)
```

Comment:
I have a problem with deprecating it. 
Why it's deprecated ? We still support l&s restore handled with RClone.

What about clusters keeping the backup on GCS ?

Reply:
We will support the restore with Rclone download+l&s, but we don't want to continue going in the direction of reporting each type of file progress in `sctool progress` (downloaded/l&s/restoredWithAPI/somethingNew bytes).
We want to focus on the thing that the user is interested in, so the amount of restored bytes or the general task progress, which is common for all restore types.
I deprecated it, but I still plan to fill this info - deprecation is just a note that we advise the user of this API to rely on the restored bytes, not downloaded bytes, which might be empty or removed in the future.

@karol-kokoszka @VAveryanov8 does this convince you?

Reply:
OK, let it be.